airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/CtidGlobalStateManager.java

```diff
@@ -144,10 +147,14 @@ public CdcState getCdcState() {
 
   }
 
+  private boolean isIncrementalStream(final AirbyteStreamNameNamespacePair pair) {
+    return !resumableFullRefreshStreams.contains(pair) && !nonResumableFullRefreshStreams.contains(pair);
```

Comment:
Thanks for adding this function!

Could we have one line of comment here to explain this?